pkg/scheduler/factory.go

```diff
@@ -600,7 +606,9 @@ func (c *Configurator) getAlgorithmArgs() (*PluginFactoryArgs, *plugins.ConfigPr
 		StorageClassInfo:               &predicates.CachedStorageClassInfo{StorageClassLister: c.storageClassLister},
 		VolumeBinder:                   c.volumeBinder,
 		HardPodAffinitySymmetricWeight: c.hardPodAffinitySymmetricWeight,
-	}, &plugins.ConfigProducerArgs{}
+	}
+
+	return factoryArgs, c.configProducerArgs
```

Comment:
I don't think we need this function at all, both factoryArgs and configProducerArgs can be members of Configurator, they get initialized when the configurator is created, and updated as we iterate over the predicates/priorities.

Reply:
Removed the function.